### ISSUE TITLE
Reporting wrong CPU usage

### ISSUE THREAD

**Issue by sky0matic** [elgatosf/streamdeck-cpu#24]
Hi there,

Is this plugin suppose to show the computer's CPU usage ?
Because if so, I think there may be an issue somewhere, because it's not reporting the correct usage.

See the screenshots below :

![image](https://user-images.githubusercontent.com/2175877/222480905-6db7612c-3cf8-49ec-a552-a1bef4f89643.png)

![image](https://user-images.githubusercontent.com/2175877/222480997-d0ea1b19-4bbc-4b3c-9148-2ffefa12d01b.png)

`OS` : Windows 11 Pro 22H2
`CPU`: AMD Ryzen 7 5800U

**Comment by jdtemple** [comment]
I'm having the same experience.

![image](https://user-images.githubusercontent.com/2552341/229291006-5743c831-8213-4fad-af82-ccb2bc327f12.png)

**Comment by Razer0123** [comment]
Same issue, used to work before i formatted

**Comment by leecollings-sb** [comment]
Same issue here also on Windows.
![image](https://github.com/elgatosf/streamdeck-cpu/assets/12139008/586aa343-62e6-4ca5-8c22-27ac35469af5)

**Comment by dwfinkelstein** [comment]
Same.  Number that appears does not come close to matching.

**Comment by fcc48** [comment]
Can confirm - same issue here

**Comment by FreemoX** [comment]
![Taskmgr_snR0X7WCDw](https://github.com/elgatosf/streamdeck-cpu/assets/7473648/82fa28f5-ec98-417a-a173-b1bc298a660a)

Just to jump on the band wagon; can confirm this is an issue.
In my case, it seems to report a value closer to its own CPU usage, rather than the total system CPU usage
![image](https://github.com/elgatosf/streamdeck-cpu/assets/7473648/85093b43-c324-491a-9d26-94dd0dc5779a)

**Comment by matthewboniface** [comment]
+1 same issue for me - elgato CPU consistently reports significantly lower than task manager 
![image](https://github.com/elgatosf/streamdeck-cpu/assets/117677379/2f02fdf2-4f8a-46dc-9bfe-8237bb71d917)

**Comment by misticx** [comment]
Same problem for me, since Rainmeter is also showing a wrong CPU usage, this might have something to do with some Windows 11 update, because both programs stopped working at the same time.

**Comment by Venjjeance** [comment]
I think @FreemoX may be correct.
My image only seems to cycle 0 - 1% and after reading through the thread, it does seem to really only show Stream Decks uses with the decimal dropped. If Stream Deck is at 1.8%, it just shows 1% / 0.7% is just 0%. In either case, this isn't working as intended, and as @misticx points out, it is definitely possibly a Win 11 update issue - I've had some other non-Stream Deck processes affected semi-recently by Win 11 updates.

Will be nice to see an update to resolve this.

**Comment by danielabbatt** [comment]
Agreed - this is really poor and pretty much useless now unfortunately.

**Comment by Allocdk** [comment]
Can confirm, it only shows its own cpu usage and not the pc cpu usage

![image](https://github.com/user-attachments/assets/8548af75-6660-471e-9242-0b548876906d)
